api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -304,13 +306,14 @@ void shouldTestClaimantBecomingIneligible(List<LocalDate> previousCycleChildrenD
         verifyNoMoreInteractions(notificationClient);
     }
 
-    //First argument is the previous cycle, second argument is the current cycle
+    //First argument is the previous cycle, second argument is the current cycle, third is the expected delivery date.
     private static Stream<Arguments> provideChildrenDobForTest() {
         return Stream.of(
-                Arguments.of(singletonList(THREE_YEAR_OLD), singletonList(THREE_YEAR_OLD)),
-                Arguments.of(emptyList(), singletonList(THREE_YEAR_OLD)),
-                Arguments.of(singletonList(THREE_YEAR_OLD), emptyList()),
-                Arguments.of(emptyList(), emptyList())
+                Arguments.of(SINGLE_THREE_YEAR_OLD, SINGLE_THREE_YEAR_OLD, EXPECTED_DELIVERY_DATE),
+                Arguments.of(emptyList(), SINGLE_THREE_YEAR_OLD, EXPECTED_DELIVERY_DATE),
+                Arguments.of(SINGLE_THREE_YEAR_OLD, emptyList(), EXPECTED_DELIVERY_DATE),
+                Arguments.of(emptyList(), emptyList(), EXPECTED_DELIVERY_DATE),
+                Arguments.of(SINGLE_THREE_YEAR_OLD, emptyList(), NOT_PREGNANT)
```

Comment:
This is the extra test where the claimant has children in the previous cycle, none in the current cycle and are not pregnant. In all of the original scenarios the claimant was pregnant.